src/libs/CardUtils.ts

```diff
@@ -80,6 +80,21 @@ function getCardDescription(cardID?: number, cards: CardList = allCards) {
     return cardDescriptor ? `${humanReadableBankName} - ${cardDescriptor}` : `${humanReadableBankName}`;
 }
 
+/**
+ * @param transactionCardName
+ * @param cardID
+ * @param cards
+ * @returns company card name
+ */
+function getCompanyCardDescription(transactionCardName?: string, cardID?: number, cards: CardList = allCards) {
```

Comment:
I think we should update these methods in the utils file to get the allCards from the component/ passed as the param to keep these functions pure. Could you update it for this method (and then we could look into doing it for other utils methods?)